### PR TITLE
Dependencies: Replace classnames with clsx - v2

### PR DESCRIPTION
**Pull request by tyxla**
## Proposed Changes

This PR replaces `classnames` with `clsx` across Calypso, which is a faster and lighter alternative. (v1 was #89843 but it was quite stale).

@DaniGuardiola introduced it in Gutenberg in https://github.com/WordPress/gutenberg/pull/61138 recently.

We can now replace it since we updated to the latest `@wordpress` packages in #91366.

## Testing Instructions

* All should be green.

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { useMediaQuery } from '@wordpress/compose';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useCallback, useEffect, useState } from '@wordpress/element';
 import { registerPlugin } from '@wordpress/plugins';
-import cx from 'classnames';
+import cx from 'clsx';
```
Comment (DaniGuardiola) [inline apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js:10]: Can we rename to clsx for consistency?

File: client/components/marketing-message/index.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import config from '@automattic/calypso-config';
 import { Button, Gridicon } from '@automattic/components';
 import styled from '@emotion/styled';
-import cx from 'classnames';
+import cx from 'clsx';
```
Comment (DaniGuardiola) [inline client/components/marketing-message/index.tsx:4]: Same as other comment. We could rename to clsx for consistency.

File: packages/design-carousel/src/item.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 import './styles.scss';
 import { getDesignPreviewUrl } from '@automattic/design-picker';
 import { MShotsImage, MShotsOptions } from '@automattic/onboarding';
-import cx from 'classnames';
+import cx from 'clsx';
```
Comment (DaniGuardiola) [inline packages/design-carousel/src/item.tsx:5]: Same as other comment. We could rename to clsx for consistency.

File: packages/site-picker/src/index.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@
 import { useFocusTrap } from '@automattic/tour-kit';
 import { __ } from '@wordpress/i18n';
 import { globe, Icon, chevronUp, chevronDown } from '@wordpress/icons';
-import cx from 'classnames';
+import cx from 'clsx';
```
Comment (DaniGuardiola) [inline packages/site-picker/src/index.tsx:5]: Same as other comment. We could rename to clsx for consistency.

File: packages/social-previews/src/nextdoor-preview/post-preview.tsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import { __ } from '@wordpress/i18n';
-import cx from 'classnames';
+import cx from 'clsx';
```
Comment (DaniGuardiola) [inline packages/social-previews/src/nextdoor-preview/post-preview.tsx:2]: Same as other comment. We could rename to clsx for consistency.

**Review by DaniGuardiola — APPROVED** [review]
LGTM! Due to the large amount of changes, and to GitHub being annoying and not loading diffs past certain amount of files, I couldn't review every single file properly. I did review every package.json file and most of the ones that GitHub did load.

I did look for more instances of the `cx` name, you'll find comments on them.

**Comment by tyxla** [comment]
Thanks @DaniGuardiola 🙌 

I'll follow-up with the rename in another PR - this PR is already huge as it is.

**Comment by tyxla** [comment]
@DaniGuardiola #91413 is a follow-up PR to address your feedback from above.

**Pull request merged**